selfdrive/car/docs_definitions.py

```diff
@@ -129,9 +125,9 @@ def get_column(self, column: Column, star_icon: str, footnote_tag: str) -> str:
     elif column == Column.MODEL and len(self.years):
       item += f" {self.years}"
 
-    footnote = get_footnote(self.footnotes, column)
-    if footnote is not None:
-      item += footnote_tag.format(self.all_footnotes[footnote])
+    footnotes = get_footnotes(self.footnotes, column)
+    if len(footnotes):
+      item += footnote_tag.format(",".join(sorted([str(self.all_footnotes[fn]) for fn in footnotes])))
```

Comment:
It's minor, but I think output readability will be improved by enclosing the list in `[]`.  Having the numbers run together with the model year is visually confusing, even if superscript is used.

![Screen Shot 2022-07-23 at 6 26 45 AM](https://user-images.githubusercontent.com/1708631/180590437-3db2d855-9955-445e-a830-12fcc5dc04e5.png)

`6,7,8` --> `[6,7,8]`

Reply:
These are AMA standard, so there's at least a standard for it: https://guides.lib.uw.edu/hsl/ama/intext

Reply:
Tried it, there's no way I found to center the stars so it didn't look too consistent:
![Screenshot from 2022-07-22 23-13-58](https://user-images.githubusercontent.com/25857203/180593017-26e6c2b2-d3c1-47c4-bd79-a4eb60f71fb5.png)